audit_log/migrations/0002_migrate_to_resilient_logger.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from datetime import datetime, timezone
+from typing import TypedDict, cast
+
+from django.apps.registry import Apps
+from django.db import migrations, models
+from resilient_logger.sources import ResilientLogSource
+
+
+class OriginalAuditEvent(TypedDict):
+    origin: str
+    status: str
+    date_time_epoch: int
+    date_time: str
+    actor: dict
+    operation: str
+    target: dict
+
+
+class OriginalMessage(TypedDict):
+    audit_event: OriginalAuditEvent
+
+
+def migrate_to_resilient_logger(apps: Apps, schema_editor):
+    AuditLogEntry: models.Model = apps.get_model("audit_log", "AuditLogEntry")
+
+    for entry in AuditLogEntry.objects.filter(is_sent=False):
+        message = cast(OriginalMessage, entry.message)
+        created_at = cast(datetime, entry.created_at).astimezone(timezone.utc)
```

Comment:
`cast` seems unnecessary here.
```suggestion
        created_at = entry.created_at.astimezone(timezone.utc)
```

Reply:
We have different kind of errors here, not sure what is going on.